hooks/useStore.js

```diff
@@ -31,23 +52,53 @@ const useStore = create(
               characters: state.characters.filter(
                 deleteCharacter => deleteCharacter.id !== id
               ),
+              toasts: state.toasts.map(toast =>
+                toast.id === 3 ? { ...toast, shown: true } : toast
+              ),
             };
           });
+          setTimeout(() => {
+            set(state => {
+              return {
+                toasts: state.toasts.map(toast => ({ ...toast, shown: false })),
+              };
+            });
+          }, 2000);
         },
 
-        toggleToast: (toastId, visible) => {
+        editCharacter: oldCharacter => {
```

Comment:
this naming is confusing, do you get here the outdated information or the new updated information about the character